index.html

```diff
@@ -541,28 +541,28 @@ <h1 class="mb-8 text-center text-[28px] font-bold leading-[34px] lg:text-[32px]
       <div class="flex justify-center">
         <ul class="grid grid-cols-2 gap-6 text-[12px] font-light leading-[20px] md:grid-cols-6">
           <li class="flex flex-col items-center justify-end">
-            <img src="./react.png" alt="React" class="h-16" />
+            <img src="./react.png" alt="" class="h-16" />
             <span class="text-center">React</span>
           </li>
           <li class="flex flex-col items-center justify-end">
-            <img src="./material-ui.png" alt="MUI" />
-            <span class="text-center">MUI</span>
+            <img src="./material-ui.png" alt="" />
```

Comment:
Fix an a11y issue, this is a decorative image, it must have an empty alt, https://www.w3.org/WAI/tutorials/images/decorative/